product/views.py

```diff
@@ -36,14 +62,16 @@ def get(self, request):
         ).values('id','shoe__id', 'name', 'price', 'main_image', 'sub_image')
         
         shoe_list = [{'product_detail' : shoe} for shoe in shoes]
+        
         for i in range(0,len(shoe_list)):
             shoe_list[i]['color_list'] = list(Color.objects.filter(**{
```

Comment:
```suggestion
        for shoe in shoe_list:
            shoe['color_list'] = list(Color.objects.filter(**{
```

for 문에서 index `i`가 꼭 필요한 경우가 아니라면 리스트를 바로 순회하여도 됩니다.
변수 이름은 가독성을 위하여 `i`가 아닌 단어로 대체해주세요